static/xref/script.js

```diff
@@ -187,6 +189,23 @@ async function ready() {
     options.all = ev.target.checked;
   });
 
+  const { searchParams } = new URL(window.location.href);
+  for (const [field, value] of searchParams) {
+    switch (field) {
+      case "term":
+      case "for":
+        form[field].value = value;
+        break;
+      case "cite":
+      case "types":
+        value.split(",").forEach(val => form[field].select(val));
+        break;
+    }
+  }
+  if (searchParams.has("term")) {
+    onSubmit();
```

Comment:
Alternative, just pass a CustomEvent?

Reply:
prevenDefault() on custom event will be a no-op.

Reply:
how about remove the event argument altogether and handle `preventDefault` at `form.addEventListener("submit", ...)`?

Reply:
Yeah, I think just send a new CustomEvent here... then the other code above doesn't need the special check.

Reply:
how about 0457ba1?